src/panels/filter-panel-react/FilterPanelControl.tsx

```diff
@@ -0,0 +1,45 @@
+import { PanelProps } from '@grafana/data'
+import { HorizontalGroup, Input, Select } from '@grafana/ui'
+import { FieldDisplay } from 'components/FieldDisplay'
+import { saveFilterPanel } from 'lib/localStorageService'
+import React, { useEffect } from 'react'
+import { FilterControlProps } from './FilterPanelTypes'
+
+export const FilterPanelControl: React.FC<PanelProps<FilterControlProps>> = (props) => {
+
+    const getSelectOptions = (filter) => {
+        // TODO: Replace these fake values with the real ones when we can get them from metricfindquery
+        return props?.options?.filterEditor?.properties?.[filter.attribute.value?.id].values ||
+            [{ value: "1", label: '1' }, { value: "2", label: '2' }]
+    }
+
+    const selectChanged = () => {
+        //TODO: When these values are real, store this selection.
+    }
+
+
+    useEffect(() => {
+        if (props.options.filterEditor) {
+            saveFilterPanel(props.options.filterEditor)
+        }
+    }, [props.options.filterEditor])
+    return (
+        <HorizontalGroup align='flex-start'>
```

Comment:
Wonder if this should be `VerticalGroup`? Fine either way for now, I'll have to work with it more to see.